Util/script.js

```diff
@@ -0,0 +1,33 @@
+import express from "express";
```

Comment:
It might be worth moving this file in to "api/util", and perhaps moving the schema def in to "api/definition" ? Would also keep the folder name lowercase for consistency.